module-bsp/board/linux/eink/LinuxEinkDisplay.cpp

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include <board.h>
+
+#include <log/log.hpp>
+
+#include <cstring>
+#include <memory>
+
+#include "LinuxEinkDisplay.hpp"
+#include "ED028TC1.h"
+
+namespace hal::eink
+{
+    EinkStatus translateStatus(const EinkStatus_e status_e)
```

Comment:
consider generic display error and implementation-specific error? :)

Reply:
I didnt wanna reinvent the wheel, especially since Im not very familiar with such drivers - since we already have all possible errors from eink implementation I decided to just translate them. In case we will add any other implementation, we will be able to create generic errors and just change the translation easily.